functions/json-loop/1.0/index.js

```diff
@@ -0,0 +1,28 @@
+const jsonLoop = async ({ input, path }, steps) => {
+  try {
+    input = JSON.parse(input);
+  } finally {
+    if (path && path.includes('.')) {
```

Comment:
you don't need the `else if`.

Just do 

```
if (path) {
  switch (true) {
    case path.includes('.'): {
      path.split('.').forEach((key) => {
        input = input[key];
      });
      break;
    }

    default:
      input = input[path];
  }
}

Reply:
Nice remark! Thank you. Updating as we speak.